lib/jpegstream.js

```diff
@@ -10,7 +10,8 @@
  * Module dependencies.
  */
 
-var Stream = require('stream').Stream;
+var Readable = require('stream').Readable;
```

Comment:
Considering the fact that most of the stream utilities depends on readable-stream, it's not really problematic (the users will do depend on it anyway). The only issue might be in the browser, but I don't think this can be used there.